src/main/java/by/custom/utilcalculator/service/Bot.java

```diff
@@ -0,0 +1,372 @@
+package by.custom.utilcalculator.service;
+
+import by.custom.utilcalculator.repository.resources.Commands;
+import by.custom.utilcalculator.repository.steps.*;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.*;
+
+public class Bot extends TelegramLongPollingBot {
+    private CountryOrigin countryOrigin = null;
+    private OwnersType ownersType = null;
+    private CarAge carAge = null;
+    private TypeOfEngine typeOfEngine = null;
+    private VolumeOfEngine volumeOfEngine = null;
+    private final CalculatorPassenger calculator;
+    private ResourceBundle bundle = null;
+
+    //singleton Bot creation
+    private static class BotHolder {
+        private static final Bot bot = new Bot();
+    }
+
+    //bot's name
+    private static final String BOT_NAME = "Бесплатный калькулятор утильсбора РБ";
+
+    //private constructor to avoid wrong bot's creation
+    private Bot() {
+        calculator = CalculatorPassenger.getInstance();
+        createBundleResources();
+    }
+
+    private void createBundleResources() {
```

Comment:
I prefer to see this logic in Main class.

Reply:
Removed it to another class: BundleResourcesServant.class